packages/wall/wall.0.4.1/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Realtime Vector Graphics with OpenGL"
+description:
+  "Lightweight, fast and declarative vector graphics rasterization using OpenGL"
+maintainer: ["frederic.bour@lakaban.net"]
+authors: ["Frédéric Bour"]
+license: "BSD3"
+homepage: "https://github.com/let-def/wall"
+doc: "https://let-def.github.io/wall/doc"
+bug-reports: "https://github.com/let-def/wall/issues"
+depends: [
+  "dune" {>= "1.11"}
+  "ocaml"
+  "gg"
+  "result"
+  "grenier"
+  "conf-gles2"
```

Comment:
Do you know if conf-gles2 can be updated to support freebsd?

Reply:
Added support for all handled distributions and OS's here: https://github.com/ocaml/opam-repository/pull/17534